### ISSUE TITLE
A quesition about the implement of FedMD and KT-pFL.

### ISSUE THREAD

**Issue by akyo1o** [Anon-Submission/FedTED#1]
Hello authors, thank you for your contribution to the implementation of Federated Distillation Learning. After I checked your code, I have a question about FedMD and KT-pFL implementation. In the code you provided, the implementation of the above two methods is very similar, one of the differences is in the aggregation function of the server. However, I read the original KT-pFL paper and I noticed that the algorithm steps in the paper don't seem to be quite the same as FedMD, and I doubt if this is a reasonable implementation? Also, FedMD should use public data and private data for pretrain in the pretrain phase, but I found in the code that you seem to use only public data for pretrain. Please forgive me if there is any misunderstanding. I look forward to hearing from you, if you are not comfortable replying in the issue, please contact me privately at cengf13h@163.com.

**Comment by HermanCloud** [comment]
Hello author, thank you for providing the implementation of the algorithm FedFTG, but I found that the implementation of the algorithm seems to be different from the original paper. When the original paper trained the generator, Loss_cls refers to the classification loss of each client, but the server is used in the code. When fine-tuning the server-side model, according to the explanation of the original paper, only model discrepancy loss should be used, and the classification loss is used to train the generator.Please forgive me if my understanding is wrong. Thank you very much for your reply.
